addons/binding/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/json/google/AddressComponent.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.icloud.internal.json.google;
+
+import java.util.ArrayList;
+
+/**
+ *
+ * @author Patrik Gfeller
+ *
+ */
+public class AddressComponent {
+    private String long_name;
```

Comment:
1) Please order such that all the fields are on top
2) please follow the naming standardsfor fields so camelcase and if needed you can use @SerializedName to link fields to another name